core/server/master/src/main/java/alluxio/master/backup/AbstractBackupRole.java

```diff
@@ -186,9 +186,9 @@ protected AlluxioURI takeBackup(BackupPRequest request, AtomicLong entryCounter)
       } catch (IOException e) {
         try {
           ufs.deleteExistingFile(backupFilePath);
-        } catch (Throwable t) {
-          LOG.error("Failed to clean up failed backup at {}", backupFilePath, t);
-          e.addSuppressed(t);
+        } catch (Exception e2) {
+          LOG.error("Failed to clean up failed backup at {}", backupFilePath, e2);
+          e.addSuppressed(e2);
         }
         throw e;
```

Comment:
Can we wrap this exception with more info, like backup path, and `entryCounter`?